davinci/src/main/kotlin/com/pingidentity/davinci/collector/SingleValueCollector.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 Ping Identity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.pingidentity.davinci.collector
+
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.jsonPrimitive
+
+/**
+ * Class representing a single value collector.
+ *
+ * This class inherits from the FieldCollector class and implements the Collector interface.
```

Comment:
_This class extends the [FieldCollector] class_

Reply:
Updated.